src/plugins/intel_gpu/include/intel_gpu/primitives/assign.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+
+#include "primitive.hpp"
+#include "intel_gpu/runtime/memory.hpp"
+
+namespace cldnn {
+
```

Comment:
Need to add comments. Please see other primitives.

/// @addtogroup cpp_api C++ API
/// @{
/// @addtogroup cpp_topology Network Topology
/// @{
/// @addtogroup cpp_primitives Primitives
/// @{

Reply:
Doxygen documentation was updated